tests/integration/test_default_attributes.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eou pipefail
+
+TARGET="detekt_with_default_attributes"
+OUTPUT_DIR="$(bazelisk info bazel-bin)/tests/integration/"
+
+bazelisk clean
+bazelisk build //tests/integration:${TARGET}
+
+test ! -f "${OUTPUT_DIR}/${TARGET}_detekt_report.html"
```

Comment:
eh, bad thing about these is that you won't get any error log, script will just exit (unsuccessfully)

but we can figure something else later, maybe even just add `set -x` before first `test -f`

Reply:
Yep, added `set -x`, makes things a bit more obvious.

Reply:
👍